scripts/feature_tests/upgrade/1cp_1w_bootDiskImage_cluster_upgrade.sh

```diff
@@ -11,20 +11,31 @@ echo '' >~/.ssh/known_hosts
 
 start_logging "${1}"
 # Provision original nodes
-set_number_of_master_node_replicas 1
+set_number_of_master_node_replicas 3
 set_number_of_worker_node_replicas 1
 
 provision_controlplane_node
+sleep 60
 
+get_target_kubeconfig
+point_to_target_cluster
 controlplane_is_provisioned
-controlplane_has_correct_replicas 1
-
-# apply CNI
-apply_cni
+controlplane_has_correct_replicas 3
 
+point_to_management_cluster
```

Comment:
like I commented in https://github.com/metal3-io/metal3-dev-env/pull/500, I would not do the kubeconfig magic in every test case but centralized in upgrade_common.sh